app/serializers/basic_group_serializer.rb

```diff
@@ -55,6 +68,62 @@ def include_automatic_membership_email_domains?
     scope.is_admin?
   end
 
+  def include_smtp_server?
```

Comment:
Can you instead add a method `admin_attributes` which would be similar to [`staff_attributes`](https://github.com/discourse/discourse/blob/1c0c425f38616e801b5b917d3c95706f9a2a8e5c/app/serializers/user_card_serializer.rb#L6-L13) in order to reduce all these lines into

```ruby
admin_attributes smtp_server, smtp_port, ....
```